### DISCUSSION TITLE
Blocked by an object

### DISCUSSION THREAD

**Discussion by TheVanhaMestari** [Courseplay/Courseplay_FS22#2522] [Q&A]
Hi, I have tried to Google around to find some help about why my Courseplay AI is getting blocked by an object all the time and won't reverse or make a turn to avoid the objects?

These things have happened for me at The Old Stream Farm and at The Valley The Old Farm maps, and I have only used vanilla equipments.

I'm receiving these errors when they are having a fence, a tree or a building in front of them or nearby at the end / sides of the fields.

So is there a way to get it sorted? I have tried to mess around with different values and settings but nothing does work for me. Of course I'm new to this mod so all help is appreciated.

**Comment by Tensuko** [comment]
That is on purpose ofc for obvious reasons.
Generate the course with round or smooth corners in that case and enough headland.
